packages/grafana-ui/src/saga-themes/tokens/core.json

```diff
@@ -0,0 +1,893 @@
+        "type": "lineHeights"
+      },
+      "2": {
+        "value": "{dimension.scale}*4",
+        "type": "lineHeights"
+      },
+      "3": {
+        "value": "{dimension.scale}*5",
+        "type": "lineHeights"
+      },
+      "4": {
+        "value": "{dimension.scale}*6",
+        "type": "lineHeights"
+      },
+      "5": {
+        "value": "{dimension.scale}*7",
+        "type": "lineHeights"
+      },
+      "6": {
+        "value": "{dimension.scale}*8",
+        "type": "lineHeights"
+      },
+      "7": {
+        "value": "{dimension.scale}*9",
+        "type": "lineHeights"
+      },
+      "8": {
+        "value": "{dimension.scale}*10",
+        "type": "lineHeights"
+      },
+      "9": {
+        "value": "{dimension.scale}*11",
+        "type": "lineHeights"
+      },
+      "10": {
+        "value": "{dimension.scale}*13",
+        "type": "lineHeights"
+      },
+      "11": {
+        "value": "{dimension.scale}*16",
+        "type": "lineHeights"
+      },
+      "12": {
+        "value": "{dimension.scale}*20",
+        "type": "lineHeights"
+      },
+      "13": {
+        "value": "{dimension.scale}*24",
+        "type": "lineHeights"
+      },
+      "14": {
+        "value": "{dimension.scale}*28",
+        "type": "lineHeights"
+      },
+      "15": {
+        "value": "{dimension.scale}*32",
+        "type": "lineHeights"
+      },
+      "16": {
+        "value": "{dimension.scale}*37",
+        "type": "lineHeights"
+      }
+    },
+    "letterSpacing": {
+      "tight": {
+        "value": "-0.25%",
+        "type": "letterSpacing"
+      },
+      "normal": {
+        "value": "0%",
+        "type": "letterSpacing"
+      },
+      "wide": {
+        "value": "0.15%",
+        "type": "letterSpacing"
+      },
+      "wider": {
+        "value": "0.25%",
+        "type": "letterSpacing"
+      }
+    },
+    "paragraphSpacing": {
+      "normal": {
+        "value": "0",
+        "type": "paragraphSpacing"
+      }
+    },
+    "textDecoration": {
+      "none": {
+        "value": "none",
+        "type": "textDecoration"
+      },
+      "underline": {
+        "value": "underline",
+        "type": "textDecoration"
+      },
+      "lineThrough": {
+        "value": "line-through",
+        "type": "textDecoration"
+      }
+    },
+    "textTransform": {
+      "none": {
+        "value": "none",
+        "type": "textCase"
+      },
+      "uppercase": {
+        "value": "uppercase",
+        "type": "textCase"
+      },
+      "lowercase": {
+        "value": "lowercase",
+        "type": "textCase"
+      },
+      "capitalize": {
+        "value": "capitalize",
+        "type": "textCase"
+      }
+    }
+  },
+  "border": {
+    "borderRadius": {
+      "default": {
+        "value": "{dimension.scale}*0.25",
+        "type": "borderRadius",
+        "description": "Standard Corner Radius used throughout Grafana."
+      },
+      "circle": {
+        "value": "100000",
```

Comment:
This should be `100%`, otherwise in code `circle` and `pill` will show exactly the same for most scenarios